code/modules/shuttles/landmarks.dm

```diff
@@ -122,6 +124,43 @@
 		if(T.density)
 			T.ChangeTurf(get_base_turf_by_area(T))
 
+//Used for custom landing locations. Self deletes after a shuttle leaves.
+/obj/effect/shuttle_landmark/temporary
+	name = "Landing Point"
+	landmark_tag = "landing"
+	flags = SLANDMARK_FLAG_AUTOSET
+	var/timer_id
+
+/obj/effect/shuttle_landmark/temporary/Initialize(var/mapload, var/shuttle_name)
+	shuttle_restricted = shuttle_name
```

Comment:
While you can set it if you like, note that your landmark won't be added to the sector it's attached to (it's added if it's in the sector's list of initial waypoints to find, or if it's /automatic), and that's the only thing this var is used for. If you do wish for it to be added, manually call `add_landmark` in here (after ..()).

Reply:
I'll go ahead and cut it, there's no real need to add it to the sector.